resources/multisite_drupal_standard.make

```diff
@@ -770,6 +770,10 @@ projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/no
 ; https://www.drupal.org/node/1084436
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-592
 projects[workbench_moderation][patch][] = patches/workbench_moderation-002-1084436-2645622-merge.patch
+; Allow cron to run workbench_moderation_set_state_action
+; https://www.drupal.org/node/2476489
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-968
+projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/workbench_moderation-reimplement-force_transaction-247648907.patch
```

Comment:
It is not the latest patch on the Drupal issue. See the PR, it uses https://www.drupal.org/files/issues/force_transition-2476489-12.patch.
Why od we have this difference? Is it a mistake or done on purpose?